core/src/main/java/de/danoeh/antennapod/core/ClientConfig.java

```diff
@@ -41,6 +42,7 @@ public static synchronized void initialize(Context context) {
         UpdateManager.init(context);
         PlaybackPreferences.init(context);
         NetworkUtils.init(context);
+        CastManager.init(context);
```

Comment:
Well, the tests work on the emulator and it doesn't have play services.  It certainly warned me when I tried to turn this on, so it's looking ok.